src/routes/solid-meta/getting-started/installation-and-setup.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Install and configure
+order: 1
+---
+
+<Callout title="Prerequisites">
+If using Solid `v1.0`, use version `0.27.x` or greater.
+For earlier versions (eg. `v0.x`), you must use `v0.26.x`.
+</Callout>
+
+## Installation
+
+To get started, install using your preferred package manager.
+
+<TabsCodeBlocks>
+<div id="npm">
+```bash frame="none"
+npm i @solidjs/meta
+```
+</div>
+
+<div id="yarn">
+```bash frame="none"
+yarn add @solidjs/meta
+```
+</div>
+
+<div id="pnpm">
+```bash frame="none"
+pnpm add @solidjs/meta
+```
+</div>
+</TabsCodeBlocks>
+
+## Setup
+
+1. Wrap your application with [`<MetaProvider />`](/solid-meta/reference/meta/metaprovider)
+2. To include head tags within your application, render any of the following:
+    1. [`<Title />`](/solid-meta/reference/meta/title): Adds the `title` of the page. 
+    2. [`<Meta />`](/solid-meta/reference/meta/meta): Adds extra metadata to the page.
+    3. [`<Style />`](/solid-meta/reference/meta/style): Adds a `style` element to the page.
+    4. [`<Link />`](/solid-meta/reference/meta/link): Specifies a relationship between the page and an external resource.
+    5. [`<Base />`](/solid-meta/reference/meta/base): Specifies the base URL for all relative URLs in the document.
+- These components can be used multiple times within the application.
+3.  If using SolidJS on the server with JSX, no additional configuration is required. 
+However, you can manage asset insertion using `getAssets` from `solid-js/web`. 
+
+Here is an example of how your code might look after this setup.
+```js
+import { MetaProvider, Title, Link, Meta } from "@solidjs/meta";
+
+const App = () => (
+	<MetaProvider>
+		<div class="Home">
+			<Title>Title of page</Title>
+			<Link rel="canonical" href="http://solidjs.com/" />
+			<Meta name="example" content="whatever" />
+		</div>
+	</MetaProvider>
+);
+```
+On the server, tags are collected, and then on the client, server-generated tags are replaced with those rendered on the client side.
+This process is important for maintaining the expected behavior, such as Single Page Applications (SPAs) when pages load that require changes to the head tags.
```

Comment:
Can you link out the package for getAssets?

```suggestion
3.  If using SolidJS on the server with JSX, no additional configuration is required:
```js
import { MetaProvider, Title, Link, Meta } from "@solidjs/meta";

const App = () => (
	<MetaProvider>
		<div class="Home">
			<Title>Title of page</Title>
			<Link rel="canonical" href="http://solidjs.com/" />
			<Meta name="example" content="whatever" />
		</div>
	</MetaProvider>
);
```
On the server, tags are collected, and then on the client, server-generated tags are replaced with those rendered on the client side.
This process is important for maintaining the expected behavior, such as Single Page Applications (SPAs) when pages load that require changes to the head tags.

If you need to manage asset insertion, you can do this using `getAssets` from `solid-js/web`. 

```